docs/tutorials/autox/autox.md

```diff
@@ -0,0 +1,198 @@
+
+## Observe experiment
+
+After starting AutoX, the HTTP SLO validation test should quickly follow. You can now use the Iter8 CLI in order to check the status and see the results of the test. 
+
+The following command allows you to check the status of the test. Note that you need to specify an experiment group via the `-g` option. The *experiment group* for experiments started by AutoX is in the form `autox-<group name>-<experiment spec name>` so in this case, it would be `autox-httpbin-iter8`.
+
+```bash
+iter8 k assert -c nofailure -c slos -g autox-httpbin-iter8
+```
+
+??? note "Sample output from assert"
+    ```
+    INFO[2023-01-11 14:43:45] inited Helm config                           
+    INFO[2023-01-11 14:43:45] experiment has no failure                    
+    INFO[2023-01-11 14:43:45] SLOs are satisfied                           
+    INFO[2023-01-11 14:43:45] all conditions were satisfied  
+    ```
+
+    We can see in the sample output that the experiment has completed and all SLOs and conditions were satisfied.
+
+And the following command allows you to check the results of the experiment.
+
+```bash
+iter8 k report -g autox-httpbin-iter8
+```
+
+??? note "Sample output from report"
+    ```
+    Experiment summary:
+    *******************
+
+    Experiment completed: true
+    No task failures: true
+    Total number of tasks: 4
+    Number of completed tasks: 4
+
+    Whether or not service level objectives (SLOs) are satisfied:
+    *************************************************************
+
+    SLO Conditions                 | Satisfied
+    --------------                 | ---------
+    http/error-count <= 0          | true
+    http/latency-mean (msec) <= 50 | true
+    
+
+    Latest observed values for metrics:
+    ***********************************
+
+    Metric                     | value
+    -------                    | -----
+    http/error-count           | 0.00
+    http/error-rate            | 0.00
+    http/latency-max (msec)    | 25.11
+    http/latency-mean (msec)   | 5.59
+    http/latency-min (msec)    | 1.29
+    http/latency-p50 (msec)    | 4.39
+    http/latency-p75 (msec)    | 6.71
+    http/latency-p90 (msec)    | 10.40
+    http/latency-p95 (msec)    | 13.00
+    http/latency-p99 (msec)    | 25.00
+    http/latency-p99.9 (msec)  | 25.10
+    http/latency-stddev (msec) | 4.37
+    http/request-count         | 100.00
+    ```
+
+    In the sample output, we can see an experiment summary, a list of SLOs and whether they were satisfied or not, as well as any additional metrics that were collected as part of the experiment.
+
+You can also produce an HTML report that you can view in the browser.
+
+```bash
+iter8 k report -g autox-httpbin-iter8 -o html > report.html
+```
+
+??? note "Sample output from HTTP report"
+    ![HTML report](images/htmlreport.png)
+
+##### Continuous and automated experimentation
```

Comment:
format of title seems wrong. too many `#`
Maybe a different title, "Modify version" or some such